qt_gui_cpp/src/qt_gui_cpp_sip/CMakeLists.txt

```diff
@@ -58,6 +91,7 @@ if(sip_helper_FOUND)
     set(LIBQT_GUI_CPP_SIP_SUFFIX ${CMAKE_SHARED_LIBRARY_SUFFIX})
   endif()
 
+  find_package(ament_cmake_python REQUIRED)
```

Comment:
This is already done the the package level CMake file.

Reply:
Removed

Reply:
In the future please reference the commit where it was removed. In this case the history was rewritten and the fix is squashed into other changes which makes it impossible to see what has changed since the last review.